src/components/organisms/pages/page_school/School.jsx

```diff
@@ -0,0 +1,46 @@
+// import { Button, Icon } from "/src/components/Organisms/pages/index.js";
+
+import { Button } from "/src/components/molecules/Button";
+import { Icon } from "/src/components/molecules/Icon";
+import { FiLock } from "react-icons/fi";
+import image from "/src/assets/teacher.jpg";
+
+export const School = ({ showCardShadow }) => {
+  const shadowClasses = showCardShadow
+    ? "lg:shadow-lg lg:rounded-3xl lg:p-16"
+    : "";
+  return (
+    <div
+      className={`flex w-[310px] flex-col items-center lg:mx-16 lg:h-[770px] lg:w-full ${shadowClasses}`}
+    >
+      <h1 className="mb-6 text-3xl font-bold">Class books</h1>
```

Comment:
Maybe it is "Teachers" instead of "Class Books"

Reply:
yes! Thanks for noticing, I also fixed in Figma where it was wrong